frontend/javascripts/admin/task/task_create_form_view.js

```diff
@@ -71,11 +71,41 @@ type State = {
   isUploading: boolean,
 };
 
-export function taskToText(task: APITask) {
+export function taskToShortText(task: APITask) {
   const { id, creationInfo, editPosition } = task;
   return `${id},${creationInfo || "null"},(${editPosition.join(",")})`;
 }
 
+export function taskToText(task: APITask) {
+  const {
+    id,
+    created,
+    dataSet,
+    type,
+    neededExperience,
+    editPosition,
+    editRotation,
+    status,
+    boundingBoxVec6,
+    projectName,
+    script,
+  } = task;
+  const neededExperienceAsString = `${neededExperience.domain},${neededExperience.value}`;
+  const [editPositionAsString, editRotationAsString] = [editPosition, editRotation].map(
+    position => `${position[0]},${position[1]},${position[2]}`,
+  );
+  const totalNumberOfInstances = status.open + status.active + status.finished;
+  const boundingBoxAsString = boundingBoxVec6
+    ? boundingBoxVec6.reduce((partialString, val) => `${partialString}${val},`)
```

Comment:
```suggestion
    ? boundingBoxVec6.join(",")
```

Or does Flow complain about this? This way you would also get rid of the trailing comma.